client/src/utils/math-jax.ts

```diff
@@ -1,15 +1,19 @@
+import { SuperBlocks } from '../../../shared/config/curriculum';
 import { scriptLoader } from './script-loaders';
 
 export const mathJaxSrc =
   'https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.4/MathJax.js?config=TeX-AMS_HTML';
 
 export const isMathJaxAllowed = (pathname: string) =>
-  [
-    '/learn/javascript-algorithms-and-data-structures-v8',
-    '/learn/project-euler/',
-    '/learn/rosetta-code',
-    '/learn/scientific-computing-with-python'
-  ].some(allowedPath => pathname.includes(allowedPath));
+  initiateMathJaxOnSuperBlock.some(superBlock => pathname.includes(superBlock));
+
+const initiateMathJaxOnSuperBlock = [
```

Comment:
```suggestion
  superBlocksWithMathJax.some(superBlock => pathname.includes(superBlock));

const superBlocksWithMathJax = [
```

Just a naming nit. `initiateMathJax` sounds like a function that initialises Mathjax, not an array of anything.